shap/explainers/_tree.py

```diff
@@ -66,38 +74,44 @@ def __init__(self, model, data = None, model_output="raw", feature_perturbation=
 
         data : numpy.array or pandas.DataFrame
             The background dataset to use for integrating out features. This argument is optional when
-            feature_perturbation="tree_path_dependent", since in that case we can use the number of training
-            samples that went down each tree path as our background dataset (this is recorded in the model object).
+            ``feature_perturbation="tree_path_dependent"``, since in that case we can use the number of training
+            samples that went down each tree path as our background dataset (this is recorded in the ``model``
+            object).
```

Comment:
Generally added code formatting in the docstrings, and some rephrasing where necessary.